src/main/java/duke/classes/Ui.java

```diff
@@ -26,79 +26,108 @@ public Ui() {
         System.out.println(welcome);
     }
 
+    /**
+     * Returns welcome message.
+     * @return String welcome message
+     */
+
+    public String welcome() {
```

Comment:
Should the Ui class be implementing these methods? As these are "response messages", I think they might be better implemented in the DialogBox class. 

I like how clean the code here is, good job on that!

Reply:
Thank you for reading my code and making the suggestions, will look towards integrating it!